aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -667,11 +667,12 @@ inline Vec256<T> operator^(const Vec256<T>& a, const Vec256<T>& b) {
 
 template<class T, typename Op>
 static inline Vec256<T> bitwise_binary_op(const Vec256<T> &a, const Vec256<T> &b, Op op) {
-  __at_align32__ intmax_t buffer[256 / sizeof(intmax_t)];
-  for (ptrdiff_t i = 0; i < sizeof(buffer) / sizeof(buffer[0]); ++ i) {
-    const intmax_t *i_a_ptr = reinterpret_cast<const intmax_t*>((const T*) a) + i;
-    const intmax_t *i_b_ptr = reinterpret_cast<const intmax_t*>((const T*) b) + i;
-    buffer[i] = op(*i_a_ptr, *i_b_ptr);
+  static constexpr auto element_no = 256 / sizeof(intmax_t);
```

Comment:
@xuhdev another question: shouldn't `element_no` (or original buffer size) be `32/sizeof(intmax_t)` (since buffer is 256 bits, i.e. 32 bytes long)

Reply:
Yes, it should be `32/sizeof(intmax_t)`. My mistake. Thanks for the catch!